cczoo/horizontal_fl/README.md

```diff
@@ -77,19 +77,22 @@ Steps **②**-**⑥** will be repeated continuously during the training process.
 
 ### Start containers and aesm services
 Start three containers (ps0, worker0, worker1) and aesm services.
+If running locally, please fill in the local PCCS server address in `<PCCS ip addr>`.
 ```shell
-./start_container.sh <attestation ip addr> ps0
-/start_aesm_service.sh
+./start_container.sh ps0 <PCCS ip addr>
+./start_aesm_service.sh
 ```
 ```shell
-./start_container.sh <attestation ip addr> worker0
-/start_aesm_service.sh
+./start_container.sh worker0 <PCCS ip addr>
+./start_aesm_service.sh
 ```
 ```shell
-./start_container.sh <attestation ip addr> worker1
-/start_aesm_service.sh
+./start_container.sh worker1 <PCCS ip addr>
+./start_aesm_service.sh
 ```
 
+If running in the cloud, please modify the `PCCS server address` in the `sgx_default_qcnl.conf` file and fill in the PCCS address of the cloud.
+
```

Comment:
Under public cloud, is <PCCS ip addr> sitll needed? Or can it be ignored as below?
./start_container.sh worker1